src/main/resources/templates/script.js

```diff
@@ -34,4 +34,11 @@ socket.onmessage = function(event){
             console.log(data[key])
         }
     }
-}
\ No newline at end of file
+}
+
+$(document).ready(function() {
+    $("#company_Symbol").autocomplete({
+        source: "dataAutoComplete",
+        minLength: 1
+    });
+});
```

Comment:
Moved this function from add.html to script.js. This jQuery function runs on page load and works perfectly.